src/config.py

```diff
@@ -51,15 +51,15 @@ class Config:
     CELERY_BROKER_URL = getenv("CELERY_BROKER_URL")
 
     # Database
-    DATABASE_NAME = getenv("DATABASE_NAME", "paralink_node")
-    DATABASE_HOST = getenv("DATABASE_HOST", "localhost")
-    DATABASE_USER = getenv("DATABASE_USER", "paralink")
-    DATABASE_PASSWORD = getenv("DATABASE_PASSWORD", "p4r4link")
-    DATABASE_URL = f"postgres://{DATABASE_USER}:{DATABASE_PASSWORD}@{DATABASE_HOST}/{DATABASE_NAME}"
+    DB_DATABASE = getenv("DATABASE_NAME", "paralink_node")
```

Comment:
Good point, I will change this back.  The reason I modified this is because it is the config format that gino-sanic expects.  I'm curious if you were you able to successfully add contracts via `/add_contract_oracle` endpoint using vanilla gino previously in place?

Reply:
Yeah, I remember having issues regarding plain `gino` vs `gino[sanic]`. I was doing some gymnastics to make it work, I do not remember exactly but could be the reason I went back to the normal `gino`. To be honest it caused many headaches, I picked it solely because it was `async`. We do not really need the uber speed. I was thinking to going to plain `SQLAlchemy` and be done with it, even more so since `async SQLAlchemy` is in beta now and could potentially bring the same benefits.

Reply:
Yes agreed, `gino[sanic]` is causing more trouble than it's worth and the speed benefits are not required.  I'll refactor to use `SQLAlchemy`.